Witch Water Improvements and Tainted Sandstone

Shnupbups opened the pull request:
- Tainted Sand can now be made into Tainted Sandstone, which has all the variants that regular or red Sandstone has
- Witch Water Buckets now have a texture and a translation
- Witch Water now gives its potion effects to every living thing except for the undead and Pumpcowns
- Witch Water now only turns Villagers into Witches if the difficulty isn't set to Peaceful
- Witch Water now turns Cows into Pumpcowns
- Witch Water now turns Overworld blocks into their Spooky Dimension equivalents when they are dropped in

Shnupbups committed:
Tainted Sandstone among other things

- Tainted Sand can now be made into Tainted Sandstone, which has all the variants that regular or red Sandstone has
- Witch Water Buckets now have a texture and a translation
- Witch Water now gives its potion effects to every living thing except for the undead and Pumpcowns
- Witch Water now only turns Villagers into Witches if the difficulty isn't set to Peaceful
- Witch Water now turns Cows into Pumpcowns
- Witch Water now turns Overworld blocks into their Spooky DImension equivalents when they are dropped in

ImAK9 reviewed (approved):
Seems fine to me! Nice!

Juuxel reviewed (changes requested):
The recipes are indented with 2 spaces, and it should be either tabs or 4 spaces.

Shnupbups commented:
Whoops, I copied them from the vanilla ones and forgot to change the indentation

Juuxel commented:
Doesn't vanilla have 4 spaces?

Shnupbups commented:
Nope, it uses 2 spaces for jsons.

Shnupbups committed:
Reformat all jsons to use tabs

All jsons now use tabs. Also added translations for biomes that were missing them.

Shnupbups committed:
more jsons

For some reason they didn't all commit last time

Juuxel commented on src/main/resources/assets/spookytime/lang/de_de.json:
```diff
@@ -1,32 +1,28 @@
+	"item.spookytime.spookium_ingot": "Spookiumbarren",
+	"item.spookytime.spookium_nugget": "Spookiumklumpen",
+	"item.spookytime.soul_bottle": "Seeleflasche",
+	"item.spookytime.pumpkin_candy": "K�rbisbonbon",
```
Did the file encoding change? These question marks were `ü` before.

Shnupbups replied:
Hmm, odd. All I did was run them through Idea's Reformat Code. I'll see if I can fix.

Shnupbups committed:
Update de_de.json
